Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -679,10 +679,11 @@ class BrowserViewController: UIViewController {
             onboarding.onboardingDelegate = self
             present(onboarding, animated: true)
         } else if BraveAds.isSupportedRegion(Locale.current.identifier) {
+            let isAdsEnabled = rewards?.ads.isEnabled ?? false
```

Comment:
Is this to check if rewards is off? If so, I'd do `rewards?.ledger.isEnabled == false`